processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -68,36 +79,103 @@ private boolean isGetterMethod(ExecutableElement method) {
         return isNonBooleanGetterName || ( isBooleanGetterName && returnTypeIsBoolean );
     }
 
+
+    /**
+     * Returns true when the {@link ExecutableElement} is a setter method. A setter starts with 'set'. The remainder of
+     * the name is supposed to reflect the property name.
+     *
+     * MapStruct will ensure that there's only one argument.
+     *
+     * @param method to be analyzed
+     * @return true when the method is a getter.
+     */
     public boolean isSetterMethod(ExecutableElement method) {
         String methodName = method.getSimpleName().toString();
 
         return methodName.startsWith( "set" ) && methodName.length() > 3;
     }
 
+    /**
+     * Returns true when the {@link ExecutableElement} is a setter method. An adder method starts with 'add'. The
```

Comment:
`... is an <em>adder</em> method.`

Do we need to add some documentation about the singularization of adder method names? (property "children", but "addChild")